.travis.yml

```diff
@@ -1,8 +1,7 @@
+sudo: false
+dist: precise
 bundler_args: --without extras
 before_script:
-  # From: https://github.com/jonleighton/poltergeist/blob/v1.0.0/.travis.yml#L5-7
-  - git submodule init
```

Comment:
This is taken care of automatically by Travis and has been for some while, so that's why I dropped these lines. Evidence is in the [logs](https://travis-ci.org/seomoz/qless/builds/305048789#L139).